package.json

```diff
@@ -209,15 +216,27 @@
     "murmurhash3js": "3.0.1",
     "ncp": "2.0.0",
     "nock": "8.0.0",
+    "node-sass": "3.8.0",
     "npm": "3.10.8",
+    "numeral": "1.5.3",
```

Comment:
Actually, I don't think it's actually being used anywhere. Thanks for catching this, I'll remove it.